provider/baked/_template_/Cargo.toml

```diff
@@ -15,3 +15,8 @@ homepage.workspace = true
 include.workspace = true
 repository.workspace = true
 rust-version.workspace = true
+
+[metadata.sources]
```

Comment:
Nit: I think it should be `[package.metadata.sources]`

https://doc.rust-lang.org/cargo/reference/manifest.html#the-metadata-table